src/App.jsx

```diff
@@ -258,6 +289,28 @@ function KanbanBoard() {
                               </Draggable>
                             ))}
                             {provided.placeholder}
+                            {user && (
+                              <div className="add-task-button-container">
+                                <button
+                                  className="add-task-button"
+                                  onClick={() => {
+                                    const taskContent = prompt('Enter task content:');
+                                    if (taskContent?.trim()) {
+                                      addTask(column.id, {
+                                        id: crypto.randomUUID(),
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
id: (() => {
  try {
    return crypto.randomUUID();
  } catch (e) {
    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
      const r = Math.random() * 16 | 0;
      return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
    });
  }
})(),
```
The crypto.randomUUID() call should be wrapped in a try-catch block as it might not be supported in all browsers. Consider providing a fallback UUID generation method.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;